does not compile in arduino IDE 2.1.0

```
/Users/tlossen/Documents/Arduino/libraries/T9602/src/T9602.cpp: In member function 'uint8_t T9602::begin(uint8_t)':
/Users/tlossen/Documents/Arduino/libraries/T9602/src/T9602.cpp:12:1: error: no return statement in function returning non-void [-Werror=return-type]
   12 | }
      | ^
/Users/tlossen/Documents/Arduino/libraries/T9602/src/T9602.cpp: In member function 'bool T9602::sleep()':
/Users/tlossen/Documents/Arduino/libraries/T9602/src/T9602.cpp:74:1: error: no return statement in function returning non-void [-Werror=return-type]
   74 | }
      | ^
cc1plus: some warnings being treated as errors

exit status 1

Compilation error: exit status 1
```

--- a/src/T9602.cpp
+++ b/src/T9602.cpp
@@ -5,7 +5,7 @@ T9602::T9602()
 {
 }
 
-uint8_t T9602::begin(uint8_t ADR_)
+void T9602::begin(uint8_t ADR_)
 {
 	ADR = ADR_;
 	Wire.begin();
@@ -71,5 +71,6 @@ String T9602::getString(bool takeNewReadings)
 bool T9602::sleep()
 {
 	//Add sleep command
+	return false;
 }
 

--- a/src/T9602.cpp
+++ b/src/T9602.cpp
@@ -13,39 +13,21 @@ void T9602::begin(uint8_t ADR_)
 
 void T9602::updateMeasurements(){
 
-  // Relative humidity
-	uint8_t data[2] = {0}; //Array for raw data from device
+	uint8_t data[4] = {0}; //Array for raw data from device
 
 	Wire.beginTransmission(ADR);
-	Wire.write(0x00);  //Read only upper 2 data bytes
+	Wire.write(0x00);
 	Wire.endTransmission();
 
-	Wire.requestFrom(ADR, 2);
-	for(int i = 0; i < 2; i++) { //Read in raw data
+	Wire.requestFrom(ADR, 4);
+	for(int i = 0; i < 4; i++) { //Read in raw data
 		data[i] = Wire.read();
 	}
 
 	// Convert RH to percent
 	RH = (float)((((data[0] & 0x3F ) << 8) + data[1]) / 16384.0) * 100.0; 
-
-
-  // Temperature
-  
-  // Reset the array to zeros
-	data[0] = 0.;
-	data[1] = 0.;
-
-	Wire.beginTransmission(ADR);
-	Wire.write(0x02);  //Read only from lower 2 data bytes
-	Wire.endTransmission();
-
-	Wire.requestFrom(ADR, 2);
-	for(int i = 0; i < 2; i++) { //Read in raw data
-		data[i] = Wire.read();
-	}
-
-	Temp = (float)((unsigned((data[2] * 64)) + unsigned((data[3] >> 2 ))) / 16384.0) * 165.0 - 40.0;  //Convert Temp
-	
+	// Convert Temp
+	Temp = (float)((unsigned((data[2] * 64)) + unsigned((data[3] >> 2 ))) / 16384.0) * 165.0 - 40.0;  
 }
 
 float T9602::getHumidity()  //Return humidity in % (realtive)

--- a/src/T9602.h
+++ b/src/T9602.h
@@ -23,7 +23,7 @@ class T9602
      *                  cannot change this, but this option exists
      *                  in case you make this change elsewhere.
 	   */
-		uint8_t begin(uint8_t ADR_ = 0x28); //use default address
+		void begin(uint8_t ADR_ = 0x28); //use default address
 
     /**
      * @brief Measure relative humidity [%] and temperature [degrees C].

